contrib/guix/libexec/build.sh

```diff
@@ -69,16 +69,12 @@ unset CPLUS_INCLUDE_PATH
 unset OBJC_INCLUDE_PATH
 unset OBJCPLUS_INCLUDE_PATH
 
-export LIBRARY_PATH="${NATIVE_GCC}/lib:${NATIVE_GCC}/lib64:${NATIVE_GCC_STATIC}/lib:${NATIVE_GCC_STATIC}/lib64"
+export LIBRARY_PATH="${NATIVE_GCC}/lib:${NATIVE_GCC_STATIC}/lib"
```

Comment:
ACK on 43c4afc407dedbafcacacfd374a24ac5f7404fb5

isn't it standard to check both lib64 and lib for libraries?

Reply:
> isn't it standard to check both lib64 and lib for libraries?

Why check for lib64 if it is known for sure that they are absent?

Reply:
Guix goes out of it's way to use `/lib` over `/lib64`, and actively patches all the packages it ships to do so. This can be revisited in future if for some reason it becomes an issue.